music.php

```diff
@@ -119,7 +119,7 @@ function tree($dir, $depth) {
 			return false;
 	}
 
-	function return_zip($name, $paths, $flat){
+	function return_zip ($name, $paths, $flat) {

```

Comment:
This is minor and completely random, but I conform to a space before the curly bracket, but not between function name and left parenthesis.

`function return_zip($name, $paths, $flat) {`